app/users/resources.py

```diff
@@ -0,0 +1,48 @@
+from flask import request
+from flask_restful import Resource
+import json
+
+from app.users.models import User
+from app import db
+
+class UserListResource(Resource):
+    def get(self):
+        users = User.query.all()
+        return {
+            'result': [{'id': user.id, 'name': user.name} for user in users]
+            }
+
+    def post(self):
+        # import pdb; pdb.set_trace()
+        new_user_data = json.loads(request.form['data'])  # this is a dictionary
```

Comment:
So would you like to use camel casing or underscores for our application? I'm fine with either. We just need to be sure to be consistent everywhere (Classes almost always follow the SomeClass convention). We typically use camel casing in our Inkling python code, but to be honest, I have always liked underscoring better.